pipelines/institution_id/02-train-model-TEMPLATE.py

```diff
@@ -373,38 +374,51 @@
                         local_path=metrics_tmp_path, artifact_path="subgroup_metrics"
                     )
 
-                    bias_flags = modeling.bias_detection.flag_bias(
-                        fnpr_data, split_name
-                    )
+                    all_subgroup_flags = modeling.bias_detection.flag_bias(fnpr_data)
+
+                    bias_subgroup_flags = [
+                        flag
+                        for flag in all_subgroup_flags
+                        if flag["flag"] not in ["🟢 NO BIAS", "⚪ INSUFFICIENT DATA"]
+                    ]
+                    for bias_flag in bias_subgroup_flags:
+                        logging.info(
+                            f"""Run {run_id}: {bias_flag["group"]} on {bias_flag["split_name"]} - {bias_flag["subgroups"]}, 
+                            FNPR Difference: {bias_flag["fnpr_percentage_difference"]}% ({bias_flag["type"]}) [{bias_flag["flag"]}]"""
+                        )
```

Comment:
this is bad form wrt logging messages -- it's strongly recommended to use "%s" instead of f-strings, one of the very few places where the old-style string interpolation is still used